assets/js/vue-apps/form-components.js

```diff
@@ -9,12 +10,20 @@ function initCharacterCount() {
     if (el) {
         new Vue({
             el: el,
+            components: {
+                'character-count': CharacterCount
+            },
             data: {
                 // Assumes a v-model on the input within the component template
                 text: null
             },
-            components: {
-                'character-count': CharacterCount
+            mounted() {
+                this.text = this.$refs.textarea.value;
```

Comment:
This fixes a bug where using `v-model` on the text area causes the input to be classed as a "managed component" so text only appears if vue knows about it. This meant that previously saved answers wouldn't show. Watching for input events and assigning the value manually avoids this.